packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2689,7 +2692,9 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
           continue;
         if (!config.isCompatibleWith(fragment.config))
           toBeMarkedExplicit.add(fragment);
-        for (final _InterestingSemanticsFragment siblingFragment in fragments.sublist(0, fragments.length - 1)) {
+        final int initialLength = fragments.length - 1;
```

Comment:
The name is a bit confusing, maybe siblingLength?

Reply:
done